language.md

```diff
@@ -22,6 +22,8 @@ Ha beszelsz magyarul, [kovess] (hungarian/malyvacukor.md)
 
 Если вы говорите по-русски, то вам [сюда](russian/zefir.md)
 
+Agar aap hindi bolte ho toh, [Hindi](Hindi/boli.md).

```

Comment:
It looks like the correct link is below, so this one should probably be removed.